apps/hosts/admin.py

```diff
@@ -1,3 +1,20 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import (ChallengeHost,
+                     ChallengeHostTeam)
+
+
+@admin.register(ChallengeHostTeam)
+class HostTeamModelAdmin(TimeStampedAdmin):
```

Comment:
My bad. I forgot to comment here. Can you please replace `HostTeamModelAdmin` with `ChallengeHostTeamModelAdmin`?